satpy/readers/yaml_reader.py

```diff
@@ -167,6 +160,7 @@ def select_files_from_directory(self, directory=None):
         for pattern in self.file_patterns:
             matching = glob.iglob(os.path.join(directory, globify(pattern)))
             filenames.extend(matching)
+            print(pattern, filenames)
```

Comment:
Should there be a print statement here ?

Reply:
No... :(